tests/migration_test.go

```diff
@@ -3052,12 +3007,7 @@ var _ = Describe("[rfe_id:393][crit:high][vendor:cnv-qe@redhat.com][level:system
 					migration := tests.NewRandomMigration(vmi.Name, vmi.Namespace)
 					_ = tests.RunMigration(virtClient, migration)
 
-					By("Expecting for an alert to be triggered")
-					eventListOpts := metav1.ListOptions{
-						FieldSelector: fmt.Sprintf("type=%s,reason=%s", k8sv1.EventTypeWarning, watch.NoSuitableNodesForHostModelMigration),
-					}
-					expectEvents(eventListOpts, 1)
-					deleteEvents(eventListOpts)
+					events.ExpectEvent(vmi, k8sv1.EventTypeWarning, watch.NoSuitableNodesForHostModelMigration)
```

Comment:
Previously we were also deleting the events. Is it correct that now we don't do it anymore?

Reply:
Yes, because we can now focus on specific objects.